tests/spec/fs.watch.spec.js

```diff
@@ -126,4 +128,4 @@ describe('fs.watch', function() {
       });
     });
   });
-});
+});
```

Comment:
One tip: it's always a good idea to end files with a newline.  We do this for cases where files get processed/concatenated together, and you want to avoid the case that the last line of one file, and the first line of the next, get joined and cause an error.